README.md

```diff
@@ -5,7 +5,7 @@ Jellyfish client SDK in React Native
 ## Installation
 
 ```sh
-npm install react-native-client-sdk
+npm install @jellyfish-dev/react-native-client-sdk
```

Comment:
there are additional (native) instructions for webrtc sdk, we should at least link them here

Reply:
Copied native instructions from the webrtc sdk